hbase-assembly/src/main/assembly/hadoop-three-compat.xml

```diff
@@ -200,16 +200,6 @@
         <include>io.opentelemetry.javaagent:*</include>
       </includes>
     </dependencySet>
-
-    <!-- Adds junit libs to lib/test -->
```

Comment:
remove these as well, need to see if need to still keep?

Reply:
I would expect that ITs need them.

Reply:
Would be handled with comment https://github.com/apache/hbase/pull/6184/files#r1799303394

Reply:
Marking as resolved!